res/layout/instant_onboarding_activity.xml

```diff
@@ -0,0 +1,129 @@
+            android:inputType="textCapWords" />
+      </com.google.android.material.textfield.TextInputLayout>
+
+    </LinearLayout>
+
+    <TextView
+        android:id="@+id/information_label"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="16dp"
+        android:layout_marginRight="16dp"
+        android:layout_marginTop="16dp"
+        android:gravity="start"
+        android:text="@string/set_name_and_avatar_explain"
+        android:textColor="@color/gray50" />
+
+    <TextView
+        android:id="@+id/privacy_policy_button"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="start"
+        android:layout_marginLeft="16dp"
+        android:layout_marginRight="16dp"
+        android:layout_marginTop="16dp"
+        android:text="@string/instant_onboarding_agree_default"
+        android:textColor="@color/delta_accent"
+        android:textSize="16sp"/>
+
+    <Button
+        style="@style/ButtonPrimary"
+        android:id="@+id/signup_button"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingLeft="16dp"
+        android:paddingRight="16dp"
+        android:layout_marginLeft="16dp"
+        android:layout_marginRight="16dp"
+        android:layout_marginTop="20dp"
+        android:layout_marginBottom="16dp"
+        android:text="@string/instant_onboarding_create"/>
+
+    <androidx.legacy.widget.Space
+        android:layout_weight="2"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"/>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+      <TextView
+          android:id="@+id/other_options_button"
+          android:layout_width="0dp"
+          android:layout_height="wrap_content"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintEnd_toStartOf="@+id/separator"
+          app:layout_constraintTop_toTopOf="parent"
+          app:layout_constraintBottom_toBottomOf="parent"
+          android:gravity="start"
+          android:padding="16dp"
+          android:text="@string/instant_onboarding_show_more_instances"
+          android:textColor="@color/gray50"/>
+
+      <TextView
+          android:id="@+id/separator"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          app:layout_constraintStart_toEndOf="@+id/other_options_button"
+          app:layout_constraintEnd_toStartOf="@+id/scan_qr_button"
+          app:layout_constraintTop_toTopOf="parent"
+          app:layout_constraintBottom_toBottomOf="parent"
+          android:paddingLeft="16dp"
+          android:paddingRight="16dp"
+          android:text=" "
+          android:textSize="16sp"
+          android:textColor="@color/gray50" />
```

Comment:
```suggestion
      <androidx.legacy.widget.Space
          android:id="@+id/separator"
          android:layout_width="wrap_content"
          android:layout_height="wrap_content"
          app:layout_constraintStart_toEndOf="@+id/other_options_button"
          app:layout_constraintEnd_toStartOf="@+id/scan_qr_button"
          app:layout_constraintTop_toTopOf="parent"
          app:layout_constraintBottom_toBottomOf="parent"
          android:paddingLeft="16dp"
          android:paddingRight="16dp" />
```

this should work the same, but is semantically more correct

Reply:
(btw, the background around why it was a TextView is because at the beginning I tried to imitate desktop and put a dot separator)